src/server/api/routers/notes.ts

```diff
@@ -0,0 +1,10 @@
+import { z } from "zod";
+
+import { createTRPCRouter, publicProcedure, protectedProcedure } from "../trpc";
+
+export const notesRouter = createTRPCRouter({
+  getNotes: publicProcedure.query(async ({ ctx }) => {
```

Comment:
Shouldn't it be privateProcedure, since only logged in users should access their notes?

Reply:
Just kept it public for now since we don't have auth setup running.

Reply:
Ok. Just keep it in mind when implementing auth.